pages/common/fish.md

```diff
@@ -0,0 +1,27 @@
+# fish
+
+> A command-line shell designed to be user firendly.
+
+- Start interactive shell:
+
+`fish`
+
+- Execute a command:
+
+`fish -c "{{command}}"`
+
+- Run commands from a file:
+
+`fish {{file.fish}}`
+
+- Check a file for syntax:
+
+`fish --no-execute {{file.fish}}`
+
+- Display version information and exit:
+
+`fish --version`
+
+- Enable debug information (ranges from 0 to 128):
```

Comment:
This seems like an advanced option. Is this necessary to be included ? Keep in mind that tldr pages should only contain commands that are used very commonly and aimed at beginners.

Reply:
It's quite useful to _me_, but is a bit higher-level than most examples, yes. removed.